### PR TITLE
[no ticket][risk=no] Update e2e tests

### PR DESCRIPTION
**Pull request by aweng98**
Minor code cleanup.

### CODE REVIEW

File: e2e/tests/conceptsets/copy-to-workspace.spec.ts [hunk 1]
```diff
@@ -41,6 +35,8 @@ async function createConceptSet(
 }
 
 describe('Copy Concept Set to another workspace', () => {
+  const destWorkspace = 'conceptsets-copyto-workspace'; // The copy-to workspace with default CDR version
```
Comment (aweng98) [inline e2e/tests/conceptsets/copy-to-workspace.spec.ts]: The destination workspace (created with default CDR version name) doesn't need to be created new per test run. Reusing the same workspace reduces number of new workspaces per run.

File: e2e/app/page/workspaces-page.ts [hunk 2]
```diff
@@ -51,7 +51,10 @@ export default class WorkspacesPage extends AuthenticatedPage {
    * Load 'Your Workspaces' page and ensure page load is completed.
    */
   async load(): Promise<this> {
-    await this.loadPageUrl(PageUrl.Workspaces);
+    const title = await this.page.title();
+    if (title !== PageTitle) {
+      await this.loadPageUrl(PageUrl.Workspaces);
+    }
```
Comment (aweng98) [inline e2e/app/page/workspaces-page.ts:57]: Check page title before load URL. It should make tests run a little faster.

File: e2e/tests/conceptsets/copy-to-workspace.spec.ts [hunk 3]
```diff
@@ -41,6 +35,8 @@ async function createConceptSet(
 }
 
 describe('Copy Concept Set to another workspace', () => {
+  const destWorkspace = 'e2eTestCopyConceptSetToWorkspace'; // The copy-to workspace with default CDR version
```
Comment (aweng98) [inline e2e/tests/conceptsets/copy-to-workspace.spec.ts:38]: The destination workspace (created with default CDR version name) doesn't need to be created new per test run. Reusing the same workspace reduces number of new workspaces per run and makes this test run faster.

File: e2e/tests/nightly/dataproc-to-gce.spec.ts [hunk 4]
```diff
@@ -14,7 +14,7 @@ describe('Updating runtime compute type', () => {
   });
 
   test('Switch from dataproc to GCE', async () => {
-    await createWorkspace(page, { cdrVersion: config.ALTERNATIVE_CDR_VERSION_NAME });
+    await createWorkspace(page, { cdrVersionName: config.ALTERNATIVE_CDR_VERSION_NAME });
```
Comment (aweng98) [inline e2e/tests/nightly/dataproc-to-gce.spec.ts:17]: Renamed parameter name `cdrVersion` to `cdrVersionName` in `test-utils` `createWorkspace` function.

**Review by freemabd — APPROVED** [review]
Looks good

**Pull request merged**